.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         id: tag_version
         run: |
           # Extracts the latest version tag, bumps the minor version
-          LATEST_VERSION=$(git tag --sort=-authordate --merged=main | head --lines 1)
+          LATEST_VERSION=$(curl -s https://api.github.com/repos/gnosis/solvers/releases/latest | jq -r '.tag_name')
```

Comment:
do we take this tag from the gnosis solvers?

Reply:
Yes, the script creates a release for the `gnosis/solvers` repo.